extern/ZAPDUtils/Utils/BitConverter.h

```diff
@@ -181,4 +181,25 @@ class BitConverter
 		std::memcpy(&value, &floatData, sizeof(value));
 		return value;
 	}
+
+	// Rewrites the rom data in-place to be in BigEndian/z64 format
+	static inline void RomToBigEndian(uint8_t* rom, size_t romSize) {
+		// Use the first byte to determine byte order
+		uint8_t firstByte = rom[0];
```

Comment:
It would probably be a good idea to check `romSize > 0` before reading from `rom`.